fjord/celery.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import
+
+import os
+
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
```

Comment:
You probably don't need this one given that in order to import this module, you first have to import `fjord/__init__.py` which already has one of these.